### DISCUSSION TITLE
Can't get proved as a student

### DISCUSSION THREAD

**Discussion by yltx** [community/community#66269] [GitHub Education]
### Hi everyone,

When I tried to get myself proved, I used my acceptance letter. But GitHub said this about my photo of it:

> Your application cannot be reviewed until you fix the following:
> You are unlikely to be verified until you have completed your GitHub user profile with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.

But I am really a student.

**Comment by arafajs** [comment]
Update your GitHub user profile to include your full name exactly as it appears on your acceptance letter. You should also add a short bio. Once you have updated your profile, log out and log back into GitHub before re-applying for verification.

- Make sure that the photo of your acceptance letter is clear and legible.
- Try uploading a different document that proves your student status, such as a transcript or a school ID.
- Contact GitHub support for assistance.

**Reply by googidaddy** [reply]
Hi, can u help me?

**Comment by yltx** [comment]
> Update your GitHub user profile to include your full name exactly as it appears on your acceptance letter. You should also add a short bio. Once you have updated your profile, log out and log back into GitHub before re-applying for verification.
> 
> * Make sure that the photo of your acceptance letter is clear and legible.
> * Try uploading a different document that proves your student status, such as a transcript or a school ID.
> * Contact GitHub support for assistance.

But where can I put my full name and where can I write a short bio?

**Reply by arafajs** [reply]
Go to: https://github.com/[Your Username] and change it.

**Reply by yltx** [reply]
![QQ图片20230907173221](https://github.com/community/community/assets/37734213/76573ed5-d864-4c43-ab8d-d37815507108)

You didn't get what I meant. I'm asking where exactly I should change,is it the blank I pointed?

**Reply by arafajs** [reply]
Yes

**Reply by yltx** [reply]
Then where can I add a short bio?
